graphene_django/tests/models.py

```diff
@@ -30,15 +34,49 @@ class Reporter(models.Model):
     pets = models.ManyToManyField('self')
     a_choice = models.CharField(max_length=30, choices=CHOICES)
 
+    reporter_type = models.IntegerField(
+        'Reporter Type',
+        null=True,
+        blank=True,
+        choices=[(1, u'Regular'), (2, u'CNN Reporter')])
```

Comment:
nit: can you add a line break before the closing `)`? just to be consistent with the rest of the changes